packages/asset-swapper/contracts/src/ERC20BridgeSampler.sol

```diff
@@ -61,14 +58,11 @@ contract ERC20BridgeSampler is
     MStableSampler,
     MooniswapSampler,
     MultiBridgeSampler,
-    NativeOrderSampler,
     ShellSampler,
     SmoothySampler,
     TwoHopSampler,
     UniswapSampler,
-    UniswapV2Sampler,
-    UniswapV3Sampler,
-    UtilitySampler
```

Comment:
`NativeOrderSampler` and `UtilitySampler` are now standalone contracts: `NativeOrderUtils` and `NetworkUtils`.